baseplate/lib/metrics.py

```diff
@@ -329,24 +367,28 @@ def __init__(self, transport: Transport, name: bytes):
         self.transport = transport
         self.name = name
 
-    def increment(self, delta: float = 1.0, sample_rate: float = 1.0) -> None:
+    def increment(
```

Comment:
Would it work to put the tags on the construction of the counter/timer/etc.? That feels more conceptually correct since the tag is part of the metric name not value. It also means we don't need `stop_tagged()` on timers since we'll just `.send()` regardless. So that'd look like this instead:

```python
batch.counter("foo", tags={"bar": "baz"}).increment()
```

Reply:
I think that makes sense!